packages/react-compose/src/next/simplifyShorthand.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { GenericDictionary } from './types';
+
+/**
+ * Ensures that the given slots are represented using object syntax. This ensures that
+ * the object can be merged along with other objects.
+ * @param props - The incoming props
+ * @param shorthandPropNames - An array of prop names to apply simplification to
+ */
+export const simplifyShorthand = (props: GenericDictionary, shorthandPropNames?: string[]) => {
+  let newProps = props;
+
+  if (shorthandPropNames && shorthandPropNames.length) {
```

Comment:
Can you please clarify why `shorthandPropNames` should be optional?

Reply:
I had a reason, but I don't think it's valid any more. Removed, but if I hit something I'll add a comment here.